Core/Debugger/WebSocket/HLESubscriber.cpp

```diff
@@ -361,15 +362,69 @@ void WebSocketHLEFuncRemove(DebuggerRequest &req) {
 	json.writeUint("size", funcSize);
 }
 
+// Remove function symbols in range (hle.func.removeRange)
+//
+// Parameters:
+//  - address: unsigned integer address for the start of the range.
+//  - size: unsigned integer size in bytes for removal
+//
+// Response (same event name):
+//  - count: number of removed functions
+void WebSocketHLEFuncRemoveRange(DebuggerRequest &req) {
+	if (!g_symbolMap)
+		return req.Fail("CPU not active");
+	if (!Core_IsStepping())
+		return req.Fail("CPU currently running (cpu.stepping first)");
+
+	u32 addr;
+	if (!req.ParamU32("address", &addr))
+		return;
+	u32 size;
+	if (!req.ParamU32("size", &size))
+		return;
+
+	if (!Memory::IsValidRange(addr, size))
+		return req.Fail("Address or size outside valid memory");
+
+	u32 first_address = g_symbolMap->GetFunctionStart(addr);
+	if (first_address == SymbolMap::INVALID_ADDRESS) {
+		first_address = g_symbolMap->GetNextSymbolAddress(addr, SymbolType::ST_FUNCTION);
+	}
+
+	u32 counter = 0;
+	for (u32 current_addr = first_address; (current_addr < addr + size) && current_addr != SymbolMap::INVALID_ADDRESS; current_addr = g_symbolMap->GetNextSymbolAddress(addr, SymbolType::ST_FUNCTION)) {
```

Comment:
This line feels awkwardly long, might be better as a while.  But I suppose it's fine.

-[Unknown]